tests/instruments/ipgphotonics/test_yar.py

```diff
@@ -0,0 +1,233 @@
+#
+# This file is part of the PyMeasure package.
+#
+# Copyright (c) 2013-2023 PyMeasure Developers
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+# THE SOFTWARE.
+#
+
+
+import pytest
+from pymeasure.test import expected_protocol
+
+from pymeasure.instruments.ipgphotonics.yar import YAR
+
+
+init_comm = [("RNP", "RNP: 0.200"), ("RMP", "RMP: 10.5"), ("RDPT", "RDPT: 0.100")]
+
+################################
+# Values according to the manual
+
+
+def test_init():
+    """This test tests already getting min/max output power."""
+    with expected_protocol(
+            YAR,
+            init_comm
+    ):
+        pass
+
+
+def test_status_emission():
+    with expected_protocol(
+            YAR,
+            init_comm + [("STA", "STA: 1")]
+    ) as inst:
+        assert inst.status == YAR.Status.EMISSION
+
+
+def test_status_backreflection():
+    with expected_protocol(
+            YAR,
+            init_comm + [("STA", "STA: 131072")]
+    ) as inst:
+        assert inst.status == YAR.Status.HIGH_BACKREFLECTION
+
+
+def test_emission_getter():
+    with expected_protocol(
+            YAR,
+            init_comm + [("STA", "STA:1")]
+    ) as inst:
+        assert inst.emission is True
+
+
+@pytest.mark.parametrize("value", (True, "ON", "on", "On", "oN"))
```

Comment:
In my opinion, we should strive for conciseness/clarity/symmetry in our interfaces. I can see the attraction of being very "general" in what to accept (and in principle that's known [a good thing](https://en.wikipedia.org/wiki/Robustness_principle)). 
However, wouldn't it be preferable to just have `True`/`False` (and the equivalent truthy things that Python accepts) as the interface for this property? 
Accepting all kinds of variations of "on" could be a slippery slope towards [YAML](https://yaml.org/type/bool.html), which accepts  
``` 
y|Y|yes|Yes|YES|n|N|no|No|NO
|true|True|TRUE|false|False|FALSE
|on|On|ON|off|Off|OFF
```
Imo, this is not something we should mimick. Curious to hear your thoughts.

Reply:
I agree: bool is sufficient.

Looking at other people's code I notice it, too, but changing my own code (predating my pymeasure time), I overlooked it... For that reason we have reviews.